src/org/usfirst/frc4904/sovereignty/strategies/GearAlign.java

```diff
@@ -16,14 +18,22 @@
 	public static double ANGLE_D = 0.015;
 	public static final double ANGLE_SETPOINT = 320;
 	public static final double ANGLE_TOLERANCE = 0.001;
-	protected AligningCamera camera;
+	protected AlignmentSystem alignSystem;
 	protected TrimmablePIDController pidController;
 	protected boolean onAngle = false;
 	private ChassisMove chassisMove;
 
-	public GearAlign(AligningCamera camera) {
-		this.camera = camera;
-		pidController = new TrimmablePIDController(GearAlign.ANGLE_P, GearAlign.ANGLE_I, GearAlign.ANGLE_D, camera);
+	public GearAlign() {
```

Comment:
These cascading constructors are pretty pointless for yearly code - let's go with just this one and move all functionality into it.